### DISCUSSION TITLE
Memory Leak

### DISCUSSION THREAD

**Discussion by AhmedSameh2020** [ZiggyCreatures/FusionCache#215] [Q&A] [answered]
Hi @jodydonetti,

Recently I've started using fusion cash in my project, but I am having a memory leak on Azure, I've set the SkipMemoryCache = true, but it still uses almost 100% of my project RAM.
![image](https://github.com/ZiggyCreatures/FusionCache/assets/29195727/57eff374-4eee-44ed-8593-5671e0b6ae21)

Here is my configuration, am I doing something wrong?

![image](https://github.com/ZiggyCreatures/FusionCache/assets/29195727/550e61ea-896c-41a6-8e4a-52be0f6afb46)

**Comment by AhmedSameh2020** [comment, marked as answer]
I found the issue: my custom Serializer wasn't disposing the Memory stream after processing data.

**Reply by jodydonetti** [reply]
Hi @AhmedSameh2020 , great, good to know it was not a memory leak on my side!
